cypress/integration/need_milk_spec.js

```diff
@@ -0,0 +1,72 @@
+describe('need milk donor results page user flow', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://secret-forest-87730.herokuapp.com/graphql', (req) => {
+      req.reply({
+        statusCode: 200,
+        fixture: 'donorMoms.json'
+      })
+    })
+
+    cy.visit('http://localhost:3000/need-milk')
+  })
+
+  it('should have a donor results container that displays three donors', () => {
+    cy.get('.main-container').should('exist')
```

Comment:
we should be consistent with use of test ids or not

Reply:
Yeah, I'm gonna add them in, just have never used them so figured I'd add them in after I finished the test!